packages.yml

```diff
@@ -3,7 +3,7 @@ packages:
 #   version: [">=0.7.0", "<0.8.0"]
 
   - git: https://github.com/fivetran/dbt_shopify_source.git
-    revision: feature/revamp/transform-enhancements
+    revision: feature/metafields
     warn-unpinned: false
-
-# - local: ../Shopify/dbt_shopify_source
+  - package: calogica/dbt_expectations
```

Comment:
i think this is a dependency in the source package

Reply:
Oh interesting 🤔 dbt_date is a dependency, but not expectations. What are your thoughts on removing this dependency from this package and just consolidating the dbt_expectations in the source package (since dbt_date is a dependency of that).

Reply:
Removed from this package and added to the source